osidb/signals.py

```diff
@@ -76,8 +88,9 @@ def populate_cvss_score(sender, instance, **kwargs):
 
 
 @receiver(pre_save, sender=Flaw)
-def update_local_updated_dt_flaw(sender, instance, **kwargs):
+def update_flaw_fields(sender, instance, **kwargs):
     instance.local_updated_dt = timezone.now()
+    update_major_incident_start_dt(instance)
```

Comment:
I would prefer to keep this separate as more functions update flaw fields.

Reply:
You mean having multiple different receivers for the `pre_save` and `sender=Flaw`?

Reply:
Oh, I overlooked that `@receiver(pre_save, sender=Flaw)` is unique - originally, I thought that `populate_cvss_score` has the same one, but actually, it has `FlawCVSS`. So my note can probably be ignored :).